content/en/docs/Kyverno CLI/_index.md

```diff
@@ -210,6 +210,107 @@ policies:
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
 
+Value files also support global values, which can be passed to all resources the policy is being applied to.
+
+Format of `value.yaml`:
+
+```yaml
+policies:
+    - name: <policy1 name>
+      resources:
+          - name: <resource1 name>
+            values:
+                <variable1 in policy1>: <value>
+                <variable2 in policy1>: <value>
+          - name: <resource2 name>
+            values:
+                <variable1 in policy1>: <value>
+                <variable2 in policy1>: <value>
+    - name: <policy2 name>
+      resources:
+          - name: <resource1 name>
+            values:
+                <variable1 in policy2>: <value>
+                <variable2 in policy2>: <value>
+          - name: <resource2 name>
+            values:
+                <variable1 in policy2>: <value>
+                <variable2 in policy2>: <value>
+globalValues:
+    <global variable1>: <value>
+    <global variable2>: <value>
+```
+
+In case of same identifier, the inner values take precedence over global values. See the pod `test-global-prod` in the following example.
```

Comment:
What does "identifier" mean?

Reply:
I'd clarify this. "Identifier" isn't in the Kyverno vocabulary that I've seen.

Reply:
Thanks @chipzoller, will do.